### ISSUE TITLE
Missing `phone_code` and `email_code` verification strategies

### ISSUE THREAD

**Issue by TmLev** [DarrenBaldwin07/clerk-rs#22]
I was trying to list all users via `clerk_rs::apis::users_api::User::get_user_list`, and the response deserialisation failed with this error:
```logs
Serde(
    Error(
        "unknown variant `email_code`, expected `admin` or `from_oauth_google`", 
        line: 1, 
        column: 754,
    )
)
```

Based on [the Backend API docs](https://clerk.com/docs/reference/backend-api/tag/Users#operation/GetUserList), they have two variants of verification:

| OTP | Admin |
| -- | -- |
| `phone_code`, `email_code` | `admin` |
| <img width="576" alt="image" src="https://github.com/DarrenBaldwin07/clerk-rs/assets/22966523/df59952e-6624-406f-9597-4c674eb1ed44"> | <img width="455" alt="image" src="https://github.com/DarrenBaldwin07/clerk-rs/assets/22966523/18d88d16-1589-42cc-a991-787743d0af12"> | 

I also can't find the `from_oauth_google` variant.

**Comment by DarrenBaldwin07** [comment]
Thanks for opening @TmLev. I'll be resolving this soon in 0.2.0 when we realign with clerk's new api spec

**Comment by DarrenBaldwin07** [comment]
This should now be fixed in `0.2.2` https://crates.io/crates/clerk-rs/0.2.2

**Comment by sjmiller609** [comment]
As of version 0.2.3 unknown variants for **strategy** and **identification_link.type** deserialize to "Other" variant. That should fix issues with listing users.
